clevertap-core/src/test/java/com/clevertap/android/sdk/MockCoreState.kt

```diff
@@ -12,9 +11,11 @@ import com.clevertap.android.sdk.validation.ValidationResultStack
 import com.clevertap.android.sdk.variables.CTVariables
 import com.clevertap.android.sdk.variables.Parser
 import com.clevertap.android.sdk.variables.VarCache
+import io.mockk.mockk
 import org.mockito.*
 
-class MockCoreState(context: Context, cleverTapInstanceConfig: CleverTapInstanceConfig) : CoreState(context) {
+// todo lp check usages and eliminate context setup
+class MockCoreState(cleverTapInstanceConfig: CleverTapInstanceConfig) : CoreState() {
```

Comment:
All of the test classes that use MockCoreState are Kotlin classes. Could we remove this implementation and use only the one in MockCoreStateKotlin ?